connectors/katalog/pkg/connector/catalog.go

```diff
@@ -6,6 +6,8 @@ import (
 	"context"
 	"encoding/json"
 
+	"fybrik.io/fybrik/connectors/katalog/pkg/v1alpha1"
+
 	"log"
 
 	connectors "fybrik.io/fybrik/pkg/connectors/protobuf"
```

Comment:
Protobuf used by an open-api version? I thought the idea was to get rid of protos and move to open-api scheme instead.